clar2wasm/src/tools.rs

```diff
@@ -95,7 +95,7 @@ impl TestEnvironment {
             .expect("Failed to insert contract analysis.");
 
         let mut contract_context = ContractContext::new(contract_id.clone(), self.version);
-        // compile_result.module.emit_wasm_file("test.wasm").unwrap();
```

Comment:
I've been leaving this commented line here because it is very helpful when debugging.

Reply:
i'm not a fan, but it's fine. Maybe add a switch or feature for this? reverted

Reply:
I agree, but it doesn't seem worth the effort to gate it for now. We can eventually delete this, or add a proper flag/gate for it.